map/api_mark_point.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "map/user_mark.hpp"
+#include "map/styled_point.hpp"
+
+class ApiMarkPoint : public StyledPoint
```

Comment:
Do we really need StylePoint base class for Bookmark and ApiMarkPoint? Where it's used?